content/hi/cloud-computing.md

```diff
@@ -0,0 +1,26 @@
+---
+title: क्लाउड कंप्यूटिंग (Cloud Computing)
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है
+
+क्लाउड कंप्यूटिंग एक ऐसा मॉडल है जो इंटरनेट पर मांग पर सीपीयू, नेटवर्क और डिस्क क्षमताओं जैसे कंप्यूट संसाधनों की पेशकश करता है।
+क्लाउड कंप्यूटिंग उपयोगकर्ताओं को दूरस्थ भौतिक स्थान में कंप्यूटिंग शक्ति तक पहुँचने और उपयोग करने की क्षमता देती है।
+क्लाउड प्रदाता जैसे ऐ डब्लयू स (AWS), जी सी पी(GCP), अज़ोर(Azure), डिजिटल ओसियन (DigitalOcean), और अन्य सभी तृतीय पक्षों की पेशकश करते हैं
+कई भौगोलिक स्थानों में संसाधनों की गणना करने के लिए किराए पर लेने की क्षमता देता है।
+
+## समस्या
+
+कंप्यूटिंग शक्ति के अपने उपयोग का विस्तार करने का प्रयास करते समय संगठनों को परंपरागत रूप से दो मुख्य समस्याओं का सामना करना पड़ा।
+वे या तो सुविधाओं का अधिग्रहण, समर्थन, डिजाइन और भुगतान करते हैं
+उनके भौतिक सर्वर और नेटवर्क को होस्ट करने या उन सुविधाओं का विस्तार और रखरखाव करने के लिए।
+क्लाउड कंप्यूटिंग संगठनों को अपनी कंप्यूटिंग जरूरतों के कुछ हिस्से को दूसरे संगठन को आउटसोर्स करने की अनुमति देती है।
+
+## यह किस तरह मदद  करता है
+
+क्लाउड प्रदाता संगठनों को कंप्यूट संसाधनों को ऑन-डिमांड किराए पर लेने और उपयोग के लिए भुगतान करने की क्षमता प्रदान करते हैं।
+यह दो प्रमुख नवाचारों की अनुमति देता है: 
+संगठन नए भौतिक बुनियादी ढांचे पर प्रतीक्षा, योजना और संसाधनों को खर्च किए बिना अपने उत्पाद या सेवा पर ध्यान केंद्रित कर सकते हैं। वे इसे अपनी जरुरत के अनुसार भड़ा [Sacale](https://glossary.cncf.io/scalability/) भी सकते है|
```

Comment:
- the correct spelling would be `बढ़ा` instead of `भड़ा`.

- please fix spelling for Scale , not [Sacale]